src/integrator/sycl/mklsycl_extensions.cpp

```diff
@@ -6,12 +6,19 @@ namespace GauXC {
 namespace sycl  {
 namespace blas  {
 
+    template <typename T>
+    void increment_kernel( const T* X, T* Y , cl::sycl::nd_item<3> item_ct) {
+        const auto tid = item_ct.get_group(2);
+        if( tid < 1 ) (*Y) += (*X);
+    }
     template <typename T> void increment(const T *X, T *Y, cl::sycl::queue *queue) {
         GAUXC_SYCL_ERROR( queue->submit([&](cl::sycl::handler &cgh) {
-           cgh.single_task([=] () {
```

Comment:
Does `single_task` not work here for some reason? That seems like a much better approach than a full blown kernel launch

Reply:
`single_task` is the appropriate one. I missed this from the revert.